src/UI/Factory.php

```diff
@@ -336,4 +335,16 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     View Controls switch between different visualisation of data.
```

Comment:
This to me feels to narrow. "Visualisation" to me sounds like what the mode view control does, but not what section view control does. I would suggest: `View Controls allow the user to explore data by modifying a given  presentation of data.`